.github/workflows/cpp_package.yaml

```diff
@@ -56,7 +56,7 @@ jobs:
               with:
                   crate: cargo-about
             - name: Prepare licenses
-              run: bash -x ../../scripts/prepare_binary_package.sh ../..
+              run: bash -x ../../scripts/prepare_binary_package.sh ${{ runner.workspace }}/cppbuild
```

Comment:
So that didn't work before?

Reply:
It produced the files, but they weren’t included in the package. Now they’re placed in the bin dir and packaged via the cmake change. The only place where this worked is slint_tool_binary.yaml :)